src/autolabel/few_shot/label_selector.py

```diff
@@ -4,6 +4,7 @@
 from collections.abc import Callable
 from typing import Dict, List, Optional, Tuple, Union
 
+import torch
```

Comment:
does base autolabel have pytorch installed? if not we should try to use numpy

Reply:
we do